shuup/admin/templates/shuup/admin/base.jinja

```diff
@@ -11,6 +11,7 @@
         <link href="{{ static("shuup_admin/img/favicon.ico") }}" rel="icon" type="image/x-icon">
         <link href="{{ static("shuup_admin/css/select2.css") }}" rel="stylesheet" type="text/css">
         <link href="{{ static("shuup_admin/css/base.css") }}" rel="stylesheet" type="text/css">
+        <link href="{{ static("shuup_admin/css/imagelightbox.css") }}" rel="stylesheet" type="text/css">
```

Comment:
if the plan is to add it to every page, we should add a bower install and let it go into vendor.css/vendor.js. Looks like a compatible fork is available here: https://github.com/rejas/imagelightbox